model/project.go

```diff
@@ -259,23 +261,70 @@ type PluginCommandConf struct {
 	// TimeoutSecs indicates the maximum duration the command is allowed to run for.
 	TimeoutSecs int `yaml:"timeout_secs,omitempty" bson:"timeout_secs"`
 
-	// Params are used to supply configuratiion specific information.
-	Params map[string]interface{} `yaml:"params,omitempty" bson:"params"`
+	// Params are used to supply configuration specific information.
+	// map[string]interface{} is stored as a JSON string.
+	Params string `yaml:"params,omitempty" bson:"params"`
```

Comment:
I think we need to leave the existing field there for backwards compatibility, and give the new things a new name.

Reply:
Is that true since we don't currently store this in the database anywhere?

Reply:
just kidding... useful for projects